LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Wellcome Collection
+Copyright (c) 2020-2022 Wellcome Collection
```

Comment:
I could never have guessed that there is [no agreed correct way](https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year) to do something straightforward with a FOSS license

Reply:
Oh, haha, I just did this to trigger the pre-commit hook, but apparently it only passes modified files to prettier. 🤷‍♀️